examples/graceful_shutdown.py

```diff
@@ -36,7 +46,12 @@ async def worker(app):
                 "time": f"Server Time : {now}",
                 "last_event_id": stream.last_event_id,
             }
-            fs.append(stream.send_json(data, id=now.timestamp()))
+            coro = send_event(
+                stream=stream,
+                data=data,
+                event_id=now.timestamp(),
+            )
```

Comment:
Feel like this comfortably sits on one line.
```suggestion
            coro = send_event(stream, data, now.timestamp())
```